ios/FluentUI.Demo/FluentUI.Demo/Demos/TabBarViewDemoController.swift

```diff
@@ -193,3 +193,59 @@ extension TabBarViewDemoController: TabBarViewDelegate {
         present(alert, animated: true)
     }
 }
+
+// MARK: - TabBarViewDemoController: DemoAppearanceDelegate
+extension TabBarViewDemoController: DemoAppearanceDelegate {
+    func themeWideOverrideDidChange(isOverrideEnabled: Bool) {
+        guard let fluentTheme = self.view.window?.fluentTheme else {
+            return
+        }
+
+        var tokensClosure: ((TabBarView) -> TabBarTokens)?
+        if isOverrideEnabled {
+            tokensClosure = { _ in
+                return ThemeWideOverrideTabBarTokens()
+            }
+        }
+
+        fluentTheme.register(controlType: TabBarView.self, tokens: tokensClosure)
+    }
+
+    func perControlOverrideDidChange(isOverrideEnabled: Bool) {
+        let tokens = (isOverrideEnabled ? PerControlOverrideTabBarItemTokens() : nil)
+        _ = tabBarView?.overrideTokens(tokens)
+    }
+
+    func isThemeWideOverrideApplied() -> Bool {
+        return self.view.window?.fluentTheme.tokenOverride(for: TabBarView.self) != nil
+    }
+
+    // MARK: - Custom tokens
+    private class ThemeWideOverrideTabBarTokens: TabBarTokens {
+        override var badgeHeight: CGFloat {
+            return .init(CGFloat(40.0))
+        }
+    }
+
+    private class PerControlOverrideTabBarItemTokens: TabBarTokens {
+        override var unselectedColor: DynamicColor {
+            return .init(
+                         light: globalTokens.sharedColors[.cornflower][.tint20], // ColorValue(0x6E6E6E) /* gray500 */,
+                         lightHighContrast: globalTokens.sharedColors[.cornflower][.tint40], // ColorValue(0x303030) /* gray700 */,
+                         dark: globalTokens.sharedColors[.teal][.tint40], // ColorValue(0x919191) /* gray400 */,
+                         darkHighContrast: globalTokens.sharedColors[.darkTeal][.tint40] /* gray200 */)
```

Comment:
Did you mean to leave the commented values in?

Reply:
I did not :)